src/platform/plugins/shared/navigation/public/plugin.test.ts

```diff
@@ -193,11 +193,28 @@ describe('Navigation Plugin', () => {
       }
     });
 
-    it('should set the feedback button visibility to "false" for deployment in trial', async () => {
+    it('should set the feedback button visibility to "false" for deployment in trial via endDate', async () => {
```

Comment:
```suggestion
    it('should set the feedback button visibility to "false" for deployment in trial', async () => {
```

Seems like we're removing knowledge of "endDate"?